lib/pry-remote.rb

```diff
@@ -139,6 +139,30 @@ def input_proxy
     end
   end
 
+  class Parser
+    def self.parse(options)
+      args = {}
+
+      opt_parser = OptionParser.new do |opts|
+        opts.on("-sSERVER", "--server=SERVER", String, "Host of the server (#{DefaultHost})") do |s|
```

Comment:
I don't know to define a default value. Can somebody help me with that?

Reply:
I think you could set a default value into the `args = {}`. So when we run the new OptionParser initialization, if we don't pass any value as args, we can fall back into the default already defined in the args hash. Something like this:

```rb
args = {
  server: '127.0.0.1',
  port: '3000'
}
```

Reply:
Good call. I was looking for a way to set the default on the `opts.on` but it seems not possible.